_pages/vot-trust-framework.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Vectors of trust
+lead: >
+  Vectors of trust can be used to specify the authentication and proofing features that are engaged for your application.
+sidenav:
+  - text: Using vectors of trust
+    href: "#using-vectors-of-trust"
+  - text: Authentication components
+    href: "#authentication-components"
+---
+
+## Using vectors of trust
+
+Vectors of trust are a combination of individual components. Each component describes a feature or requirement for the user during the transaction. For example, a component may specify that identity-proofing or that a particular MFA method must be used.
+
+Vector components are made up of a upper-case letter followed by a lower-case letter or a number. For example, `P1` or `C2`. A vector of trust is made by combining these components and arranging them into a period-separated list. For example, `C1.C2.P1.Pb`.
```

Comment:
Suggestion: the example "P1 or C2" shows the same combination, perhaps we should make it a letter and number and a letter and lower case letter.

Reply:
I just changed it to '"P1" or "Ca"'